cartridges/int_cloudinary/cartridge/templates/default/content/renderContentImage.isml

```diff
@@ -5,7 +5,7 @@
 <isset name="imageJSON" value="${pdict.imgURL}" scope="page"/>
 <isif condition="${!empty(imageJSON.url)}">
     <img
-        alt="${pdict.altText}"
+        alt="${pdict.altText || imageJSON.url}"
```

Comment:
@asad-rafter Was it tested? The issue was that the altText from BM should override the URL, so this change doesn't look valid

Reply:
@asad-rafter ^

Reply:
It seems to be working on 007, I have verified there.

Reply:
ok, Thanks